ui/app/styles/core.scss

```diff
@@ -33,7 +34,6 @@
 @import './core/menu';
 @import './core/message';
 @import './core/navbar';
-@import './core/notification';
```

Comment:
double/triple checked. This was not being used at all. However, there is a big chunk of bulma css for notifications. I suspect we once used bulma notifications, and override some of the style in this file. But we don't use them now.